src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs

```diff
@@ -11,11 +11,10 @@ internal static IEnumerable<ExtendedTypeDefinition> GetFormatData()
         {
```

Comment:
@powercode, your last commit had 2 failures in `PowerShell-CI-windows`
Get-ItemProperty.Should be able to access a property using the Path and name switches
```powershell
Expected $null, but got C.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-ItemProperty.Tests.ps1: line 36
36: 	$output.PSDrive | Should -Be $testprovider
```
String cmdlets.Select-String.Network path
```powershell
Expected 2, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\string.tests.ps1: line 56
56:             (select-string -LiteralPath $fileNameAsNetworkPath "b").count | Should -Be 2
```